Index: linux-4.14.187/drivers/net/Kconfig
===================================================================
--- linux-4.14.187.orig/drivers/net/Kconfig
+++ linux-4.14.187/drivers/net/Kconfig
@@ -485,4 +485,6 @@ config FUJITSU_ES
 
 source "drivers/net/hyperv/Kconfig"
 
+source "drivers/net/rtl819x/Kconfig"
+
 endif # NETDEVICES
Index: linux-4.14.187/drivers/net/Makefile
===================================================================
--- linux-4.14.187.orig/drivers/net/Makefile
+++ linux-4.14.187/drivers/net/Makefile
@@ -75,3 +75,5 @@ obj-$(CONFIG_HYPERV_NET) += hyperv/
 obj-$(CONFIG_NTB_NETDEV) += ntb_netdev.o
 
 obj-$(CONFIG_FUJITSU_ES) += fjes/
+
+obj-$(CONFIG_RTL_819X_SWCORE) += rtl819x/
